packages/ibm-products/src/components/TagSet/TagSet.tsx

```diff
@@ -161,6 +165,7 @@ export let TagSet = React.forwardRef<HTMLDivElement, TagSetProps>(
       allTagsModalSearchLabel,
       allTagsModalSearchPlaceholderText,
       showAllTagsLabel,
+      disableOverflowPopup = false,
```

Comment:
this can have an undefined value, since !undefined will be false anyway, in the render beneath.

Reply:
I think this is fine in this case, undefined would only be an issue if the user explicitly defines it as undefined. And since undefined is often associated with false, it wont be an issue in this case.